README.md

```diff
@@ -311,6 +311,28 @@ Period Range
 ```csharp
  .InRelativePeriodRange(Period.FromWeeks(2), Period.FromDays(20))
 ```
+ 
+### UnitOfMeasure conversion for Actual and Versioned Time Series
```

Comment:
Lack of description of UnitOfMeasure supported ISO.

Much more documentation is needed here.

Reply:
More documentation is needed.

This documentation doesn't explain

- How to register with reference to the syntax for UoM combinations like liter/s (flow rates) and Prices, including the 'cents' which are common 

- Supported Convertions

@arkcecchi please support in authoring the specification

I expect a Unit of Measure section with example of how to Register, how to Convert (this one) and UoM syntaxt